tests/data/sample_fragments/template_without_parameter_section.json

```diff
@@ -5,6 +5,7 @@
         "S3Bucket": {
             "Type": "AWS::S3::Bucket",
             "DeletionPolicy": "Retain",
+            "UpdateReplacePolicy": "Retain",
```

Comment:
This change is needed to bring coverage to 100% and not have to write an extra test